index.html

```diff
@@ -1418,6 +1461,9 @@
           </ul>
         </li>
         <li>
+          Let <var>options</var> be the second argument.
+        </li>
+        <li>
           Let <var>timeout</var> be the value of <var>options.timeout</var> if
```

Comment:
You can and should specify default values in the definition of [NFCPushOptions](https://rawgit.com/zolkis/web-nfc/gh-pages/index.html#idl-def-NFCPushOptions) (the argument type), instead of introducing local variables in the algorithm.